### DISCUSSION TITLE
Jest continues to run after testing is completed

### DISCUSSION THREAD

**Discussion by edustrivelabs** [testcontainers/testcontainers-node#610] [Q&A] [answered]
### Issue Description
I am implementing some tests with Jest that use a testcontainer. The management with the testcontainer is working fine (setup and teardown) and every test is passing, but for some reason Jest is not terminating the run when tests are done. It prints a message saying _Jest did not exit one second after the test run has completed_.

Using the combination of `--forceExit` and `--detectOpenHandles` options for Jest command prevents this behavior but this affects performance.

I wonder if there is some way to avoid this without using those Jest parameters. I am attaching a local execution screenshot.

### Configuration
| Dependency | Version |
|--------------|---------|
| testcontainers | 9.10.1 |
| jest | 29.3.1 |
| ts-jest | 29.0.3 |
| typescript | 4.7.4 |
### Screenshot
![screenshot](https://github.com/testcontainers/testcontainers-node/assets/102308783/9638f718-5dee-45dc-bed7-7f4081b25da2)

**Comment by cristianrgreco** [comment]
Hi @edustrivelabs, I suspect you are not stopping a resource (container/network) before your test suite ends. Without logs or reproducible code I cannot help much more. I assume this is a usage issue as this issue is not seen in this project's pipelines.

**Reply by edustrivelabs** [reply, marked as answer]
Thanks for the reply @cristianrgreco! I need to apologize. We found the reason of our bug. It is related to a dependency that we are using which is called [pinojs](https://github.com/pinojs/pino).
